scripts/build/Platform/Windows/build_config.json

```diff
@@ -36,6 +36,28 @@
       "test_cpu_profile_vs2019"
     ]
   },
+  "periodic_test_profile_vs2019_pipe": {
+    "TAGS": [
+      "nightly-incremental",
+      "nightly-clean"
+    ],
+    "steps": [
+      "aws_integration_deployment",
+      "periodic_test_profile_vs2019"
+    ]
+  },
+  "aws_integration_deployment": {
+    "TAGS": [
+      "weekly-build-metrics"
+    ],
+    "COMMAND": "deploy_cdk_applications.cmd",
+    "PARAMETERS": {
+      "O3DE_AWS_PROJECT_NAME": "AWSAUTO",
+      "O3DE_AWS_DEPLOY_REGION": "us-west-2",
+      "ASSUME_ROLE_ARN": "arn:aws:iam::645075835648:role/o3de-automation-tests",
+      "COMMIT_ID": "HEAD"
```

Comment:
I think we may need to lock cdk commit here as well? instead of HEAD?

Reply:
This is a build parameter people can change. By default, we want to use the latest unless someone specify a commit ID.

Reply:
Minor: I think we want to lock it to specific commit for cdk deployment which we have discussed in the infra team meeting. (we should pick a commit id)

Reply:
We can address this in a future PR once the Jenkins job is set up. We will find a stable version (currently the latest should be stable) and lock it.